src/python/turicreate/test/test_drawing_classifier.py

```diff
@@ -290,8 +290,6 @@ def test_export_coreml(self):
             filename = _mkstemp("bingo.mlmodel")[1]
             model.export_coreml(filename)
 
-    # @unittest.skipIf(_sys.platform != "darwin", "Core ML only supported on Mac")
-    @pytest.mark.xfail(IS_PRE_6_0_RC, reason="Coming soon: Need to test and debug export_to_coreml")
     def test_export_coreml_with_predict(self):
```

Comment:
You commented out the `@unittest.skipIf(_sys.platform != "darwin", "Core ML only supported on Mac")`. You need to add it back. coremltools doesn't support predict on Linux.

Reply:
good

Reply:
I did remember that I installed on my linux box before. And now I verified it:
```
coremltools-3.1-cp36-none-manylinux1_x86_64.whl (3.4MB)
```
it supports `manylinux1`.

Reply:
The wheel is supported but the actual predict call in coremltools is only possible on Mac. So the "test_with_predict" only will work on Mac.

Reply:
I'm afraid this is not the case. It passed all our internal Linux platform pipeline.

Reply:
per offline discussion. It's my mistake of overlooking the build log.